src/basic.jl

```diff
@@ -16,14 +16,9 @@ function BasicSemialgebraicSet(V::AlgebraicSet{T, PT, A, ST}, p::Vector{PS}) whe
     BasicSemialgebraicSet(convert(AlgebraicSet{U, PU, A, ST}, V), Vector{PU}(p))
 end
 #BasicSemialgebraicSet{T, PT<:APL{T}}(V::AlgebraicSet{T, PT}, p::Vector{PT}) = BasicSemialgebraicSet{T, PT}(V, p)
-function basicsemialgebraicset(V, p)
```

Comment:
Why removing this ? There is no need to introduce this breaking change

Reply:
Sorry, I didn't notice it was exported. It didn't get hit in tests.